src/interfaces/classes.ts

```diff
@@ -0,0 +1,43 @@
+// Copyright 2020 OpenStax Poland
+// Licensed under the MIT license. See LICENSE file in the project root for
+// full license text.
+
+import { dedup } from '../util'
+
+/** Elements which may optionally have classes */
+export interface WithClasses {
```

Comment:
This should extends `Element` interfaces, right?

Reply:
Or interfaces of elements which may contain classes

Reply:
I intended `WithClasses` to be used as a mixin, and never as an element by itself.